src/Compiler/AbstractIL/il.fs

```diff
@@ -1985,7 +2018,7 @@ type ILMethodDef
 
     member x.IsZeroInit = x.MethodBody.IsZeroInit
 
-    member md.CallingSignature =
+    member md.GetCallingSignature() =
```

Comment:
Wouldn't make it lazy make more sense? In case we need to access it multiple times, we don't compute it multiple times?

Reply:
It's just a record-creating no-op, and only used when a match by name is already found.
Better to keep the cost just in time, rather then allocate additional lazy<> each time (even when not used in the end)